main.go

```diff
@@ -39,6 +38,13 @@ var (
 	errParseECPoint  = "failed to parse EC point"
 )
 
+const (
+	// January 1, 2022
+	firstEpochTimestampMs int64 = 1640995200000
```

Comment:
Any reason for this to be ms instead of seconds?

Reply:
> What do you think about RFC 3339 for the timestamps in the `/info` response instead of an integer? I find that easier to reason about looking at traffic logs.

@rillian totally agree. do you know if `FromString` in `base::Time` supports the format?

Reply:
I'm not sure what you mean by extra precision? We're rotating the epoch on a second boundary, and it's hard to imagine needing more precision than that. Anything sub-second will be perturbed by clock skew and network latency between the server and the client.

Simplifying the client is more persuasive. What does your client code look like? Chromium [uses](https://source.chromium.org/chromium/chromium/src/+/main:base/time/time.h;l=5-11) int64 microseconds since 1601-01-01T00:00:00Z, which certainly isn't directly comparable. But I'd expect clients to use some parser to convert whatever's in the response to an internal timestamp representation anyway.

Reply:
Also note that the full `int64` range isn't necessary representable in json; most javascript engines use a double internally. There should still be enough precision for this application though.

Reply:
I was thinking more about being able to compare current time ms to the next epoch ms without using the `base::Time` struct (potentially). Now that we're using RFC3339, I changed the constant to a parsed date.